### PR TITLE
Satisfies #522

### PR DESCRIPTION
**Pull request by collint8**
Satisfies #522

### CODE REVIEW

File: Superbuild/Superbuild.cmake [hunk 1]
```diff
@@ -98,6 +102,7 @@ IF(TRAVIS_BUILD)
     SET(DOWNLOAD_TOOLKITS OFF)
     SET(BUILD_HEADLESS ON)
     SET(BUILD_WITH_PYTHON OFF)
+  SET(BUILD_WITH_SCIRUN_DATA OFF)
```
Comment (aylakhan) [inline Superbuild/Superbuild.cmake]: Consistent formatting would be nice.

File: Superbuild/SCIRunDataExternal.cmake [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# 
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+#  OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#  THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+
+SET_PROPERTY(DIRECTORY PROPERTY "EP_BASE" ${ep_base})
```
Comment (aylakhan) [inline Superbuild/SCIRunDataExternal.cmake]: Should look more like DataExternal.cmake.

File: Superbuild/Superbuild.cmake [hunk 3]
```diff
@@ -98,6 +102,7 @@ IF(TRAVIS_BUILD)
     SET(DOWNLOAD_TOOLKITS OFF)
     SET(BUILD_HEADLESS ON)
     SET(BUILD_WITH_PYTHON OFF)
+    SET(BUILD_WITH_SCIRUN_DATA OFF)
   ELSE()
     # try building everything with clang!
```
Comment (dcwhite) [inline Superbuild/Superbuild.cmake]: @collint8 Turn off the flag for clang too by adding it here, in the ELSE(): 
`SET(BUILD_WITH_SCIRUN_DATA OFF)`

File: Superbuild/SCIRunDataExternal.cmake [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+SET(data_SVN_URL "https://gforge.sci.utah.edu/svn/CIBCData")
+SET(sci_data_DIR "${CMAKE_BINARY_DIR}/SCIRunData")
+
+ExternalProject_Add(SCI_data_external
+  SVN_REPOSITORY "${data_SVN_URL}"
+  SVN_TRUST_CERT 1
+  SVN_USERNAME "anonymous"
+  SVN_PASSWIRD ""
```
Comment (dcwhite) [inline Superbuild/SCIRunDataExternal.cmake]: Did you mean SVN_PASSWORD???

**Pull request merged**